docs/contracts/events.rst

```diff
@@ -73,7 +73,7 @@ four indexed arguments rather than three.
     In particular, it is possible to "fake" the signature of another event
     using an anonymous event.
 
-::
+.. code-block:: solidity
```

Comment:
Do we only support this kind now? Can't we make the extraction more flexible? (the sphinx settings say that "solidity" is the default language if none is given)

Reply:
> Do we only support this kind now? 

With this PR yes. I can add `::` as well, I don't think that pose problems

Reply:
Please also add `::` - the other one is just to "make it look nice on github".